db/config.cc

```diff
@@ -1363,6 +1363,7 @@ db::config::config(std::shared_ptr<db::extensions> exts)
     // alternator_max_items_in_batch_write matches DynamoDB behaviour of size limit, but with different value - for DynamoDB it's 25
     // (see DynamoDB's documentation for BatchWriteItem command)
     , alternator_max_items_in_batch_write(this, "alternator_max_items_in_batch_write", value_status::Used, 100, "Maximum amount of items in single BatchItemWrite call.")
+    , alternator_max_expression_cache_entries(this, "alternator_max_expression_cache_entries", liveness::LiveUpdate, value_status::Used, 0, "Maximum amount of entries in expression cache.")
```

Comment:
0 means it will never be used. No one will edit the configuration.

Reply:
I think you should also add a clarification that this is per-shard (how did we explain other per-shard numbers in the config?)